include/boost/static_string/static_string.hpp

```diff
@@ -5614,6 +5614,33 @@ operator+(
     std::size_t(0), +lhs);
 }
 
+//------------------------------------------------------------------------------
+//
+// erase_if
+//
+//------------------------------------------------------------------------------
+
+template<
+    std::size_t N, typename CharT,
+    typename Traits, typename UnaryPredicate>
+BOOST_STATIC_STRING_CPP14_CONSTEXPR
+typename // required for C++11
```

Comment:
We don't quite need this comment, but this is just a nitpick.

Reply:
Done